### DISCUSSION TITLE
Trying to build/run on Mac M1... and it's a complete and utter failure

### DISCUSSION THREAD

**Discussion by donangel** [Fi0x/EDCT#102] [General]
Hi!

I'm trying to get this tool running on my Mac M1 Pro.

I failed running the binary, so I tried the sources. Can't even get them compiled correctly (IntelliJ community edition).

Can someone help with an advice? Another level of complexity is that I'm a C++ engineer, so Java is quite far from me in this sense.

Thanks in advance for anyone, who'd be willing to help!

**Comment by Fi0x** [comment]
I'm not sure if it will work on a Mac, but IntelliJ should make it pretty easy to setup the required stuff.
After cloning the repo, you should see a Maven-Tab in IntelliJ where you can run the `install` task. This should setup everything you need to run the project.

After installing all the Maven-stuff, you should be able to run the Main-class. Use the `-d` and `-v` arguments to get more console-output for debugging.

Unfortunately the tool is currently not working at all, since **inara** changed some URLs that are required to initialize the database. If you're interested in fixing this, please let me know and create a pull-request, so that I can release a newer version for everyone.
